install.sh

```diff
@@ -155,6 +155,10 @@ echo "==== Create $env_name conda environment"
 
 conda env create --name "$env_name" --force --file="$conda_env_file"
 
+# Avoid a 'frankenconda' env that uses Python from the base env.
+# See https://github.com/illinois-ceesd/emirge/pull/132 for details.
+conda deactivate
+conda deactivate
```

Comment:
Why not do this move before the first `activate`?

Reply:
Because we need a `conda` command available to be able to deactivate.  My hypothesis on what's happening:

- `source miniforge/bin/activate` doesn't set all env variables correctly (`$PATH` ?)
- further `conda activate foobar` commands do not fix that
- when deactivating conda fully, (env+base), future activations untwist the env variables